dosocs2/scanners/nomos.py

```diff
@@ -13,20 +16,29 @@ def __init__(self, conn, config):
         super(Nomos, self).__init__(conn, config)
         self.exec_path = config['scanner_' + Nomos.name + '_path']
         self.search_pattern = re.compile(r'File (.+?) contains license\(s\) (.+)')
+        lics = schema.licenses.alias()
```

Comment:
Better to cut out the check for official license status since it adds unnecessary overhead